ddtrace/contrib/tornado/stack_context.py

```diff
@@ -26,9 +26,6 @@ class TracerStackContext(DefaultContextProvider):
         Every time a new manager is initialized, a new ``Context()`` is created for
         this execution flow. A context created in a ``TracerStackContext`` is not
         shared between different threads.
-
-        This implementation follows some suggestions provided here:
-        https://github.com/tornadoweb/tornado/issues/1063
```

Comment:
this one is still relevant I think? Did you think otherwise?

Reply:
Not sure why I removed this one, will undo this 😓